pkg/infra/iac3/resource_template.go

```diff
@@ -242,6 +254,33 @@ func exportsNodeToTemplate(tc *TemplatesCompiler, tmpl *ResourceTemplate, node *
 	return exportsTemplates, errs
 }
 
+func importFuncNodeToTemplate(node *sitter.Node, name string) (*template.Template, string, error) {
+	importFunc := doQuery(node, findImportFunc)
+	imp, found := importFunc()
+	if !found {
+		return nil, "", nil
+	}
+	outputType := imp["return_type"].Content()
+	var expressionBody string
+	if outputType == "void" {
```

Comment:
Does import returning `void` make any sense? What's the point of importing if it's not used. If it's used, it must not be `void`.